webapp/client/src/pages/HasDisabilityPersonAPage.js

```diff
@@ -17,7 +17,7 @@ const DetailsDiv = styled.div`
   }
 `;
```

Comment:
Unfortunately I am not able to select the lines. However, I am wondering if we should put that DetailsDiv directly into the Details component or add a different details component that is not used in the label somewhere?

```javascript 
const DetailsDiv = styled.div`
  margin-bottom: var(--spacing-04);
  @media (max-width: 500px) {
    margin-bottom: 0;
   }
  `;
```

Reply:
Mh, in my head the margin was more an exception. But we're using it on three different pages now (at least we should), so probably making this a new component could make sense.

Reply:
That's why I thought about it, too.

Reply:
I have added a component for that